server/src/main/java/io/seata/server/ParameterParser.java

```diff
@@ -0,0 +1,116 @@
+/*
+ *  Copyright 1999-2019 Seata.io Group.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package io.seata.server;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+
+/**
+ * The type parameter parser
+ *
+ * @author xingfudeshi@gmail.com
+ * @date 2019/05/30
+ */
+public class ParameterParser {
+    private static final String PROGRAM_NAME = "sh seata-server.sh";
```

Comment:
what for windows？

Reply:
> what for windows？

I think we can simply write seata-server,and the final result will be like:
![image](https://user-images.githubusercontent.com/24457941/58621367-c54beb80-82fb-11e9-9af3-3490641874f9.png)
in addition,it's not necessary to detect the OS for this.

Reply:
May be sh seata-server.sh(for linux and mac) or cmd seata-server.bat(for windows) [options]

Reply:
ok.

Reply:
![image](https://user-images.githubusercontent.com/24457941/58674739-bdd02500-8383-11e9-815b-a656b835e4c0.png)